crates/fj-host/src/host.rs

```diff
@@ -16,7 +16,10 @@ impl Host {
     pub fn from_model(model: Model) -> Result<Self, Error> {
         let watch_path = model.watch_path();
         let evaluator = Evaluator::from_model(model);
-        let _watcher = Watcher::watch_model(&watch_path, &evaluator)?;
+        let _watcher = match Watcher::watch_model(&watch_path, &evaluator) {
+            Ok(_watcher) => _watcher,
+            Err(e) => return Err(e),
+        };
```

Comment:
What is this change about? It doesn't seem to make a difference, except make the code longer?

Reply:
You're right. This was needed when `from_model` function was returning an `Option<Result<Self, Error>>` but I forgot to change this to the original when converting it back to returning a `Result<Self, Error>`.

Reply:
Ah, got it. Thanks for the follow-up PR!